_config.php

```diff
@@ -38,7 +37,7 @@
             . "tbody[id|class|style],thead[id|class|style],tfoot[id|class|style],"
             . "#td[id|dir|class|colspan|rowspan|width|height|align|valign|scope|style],"
             . "-th[id|dir|class|colspan|rowspan|width|height|align|valign|scope|style],caption[id|dir|class],"
-            . "-div[id|dir|class|align|style],-span[class|align|style],-pre[class|align],address[class|align],"
+            . "-div[id|dir|class|align|style|data*],-span[class|align|style],-pre[class|align],address[class|align],"
```

Comment:
This is needed for embeds. It looks like data attributes were being implicitly allowed in the past, somhow... I need to see whether that was intentional and what we need to do about it (i.e. do we always allow them or just add in the changelog that not all elements allow them by default?)

Reply:
That was incorrect - I hadn't noticed the shortcodes for embed and media were being stored in their temporary HTML format.